azure-spring-cloud-maven-plugin/src/main/java/com/microsoft/azure/maven/spring/spring/SpringAppClient.java

```diff
@@ -122,4 +127,13 @@ public AppResourceInner getApp() {
     public String getAppName() {
         return appName;
     }
+
+    private boolean isEnablePersistentStorage(SpringConfiguration configuration) {
+        return configuration != null && configuration.getDeployment() != null && configuration.getDeployment().isEnablePersistentStorage();
```

Comment:
is configuration.getDeployment().isEnablePersistentStorage() returns Boolean or boolean?

Reply:
It's a Boolean value generated by `BooleanUtils.isTrue()`